### ISSUE TITLE
Incompatible library version: PolarBleSdk and RxSwift

### ISSUE THREAD

**Issue by avik-efcom** [polarofficial/polar-ble-sdk#184]
***Platform your question concerns:*** 
- [ ] Android 
- [X] iOS
- [ ] Other
- [ ] Platform is not relevant for this question

***Device:***
- [ ] Polar OH1
- [ ] Polar Verity Sense
- [ ] Polar H10
- [ ] Polar H9
- [ ] Other
- [X] Device is not relevant for this question

**Description:**
Hi everyone,
I try to work use the SDK and as the guide say I need to download RxSwift(6.2.0)
when I start run the project I got this message

```
dyld: Library not loaded: @rpath/RxSwift.framework/RxSwift
 Referenced from: /private/var/containers/Bundle/Application/DCFDE6CE-2C32-48A2-BB5A-61B33552082E/XXX.test.app/Frameworks/PolarBleSdk.framework/PolarBleSdk
 Reason: Incompatible library version: PolarBleSdk requires version 1.0.0 or later, but RxSwift provides version 0.0.0
```
I run this command on the xcframework folder
`otool -L RxSwift.xcframework/ios-arm64_armv7/RxSwift.framework/RxSwift`
and got this output

```
RxSwift.xcframework/ios-arm64_armv7/RxSwift.framework/RxSwift (architecture armv7):
	@rpath/RxSwift.framework/RxSwift (compatibility version 0.0.0, current version 0.0.0)
	/System/Library/Frameworks/Foundation.framework/Foundation (compatibility version 300.0.0, current version 1774.101.0)
	/usr/lib/libobjc.A.dylib (compatibility version 1.0.0, current version 228.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1292.60.1)
	@rpath/libswiftCore.dylib (compatibility version 1.0.0, current version 1200.2.41)
	@rpath/libswiftCoreFoundation.dylib (compatibility version 1.0.0, current version 1.6.0, weak)
	@rpath/libswiftCoreGraphics.dylib (compatibility version 1.0.0, current version 2.0.0, weak)
	@rpath/libswiftDarwin.dylib (compatibility version 1.0.0, current version 0.0.0, weak)
	@rpath/libswiftDispatch.dylib (compatibility version 1.0.0, current version 4.40.2)
	@rpath/libswiftFoundation.dylib (compatibility version 1.0.0, current version 20.0.0)
	@rpath/libswiftObjectiveC.dylib (compatibility version 1.0.0, current version 1.0.0, weak)
RxSwift.xcframework/ios-arm64_armv7/RxSwift.framework/RxSwift (architecture arm64):
	@rpath/RxSwift.framework/RxSwift (compatibility version 0.0.0, current version 0.0.0)
	/System/Library/Frameworks/Foundation.framework/Foundation (compatibility version 300.0.0, current version 1774.101.0)
	/usr/lib/libobjc.A.dylib (compatibility version 1.0.0, current version 228.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1292.60.1)
	@rpath/libswiftCore.dylib (compatibility version 1.0.0, current version 1200.2.41)
	@rpath/libswiftCoreFoundation.dylib (compatibility version 1.0.0, current version 1.6.0, weak)
	@rpath/libswiftCoreGraphics.dylib (compatibility version 1.0.0, current version 2.0.0, weak)
	@rpath/libswiftDarwin.dylib (compatibility version 1.0.0, current version 0.0.0, weak)
	@rpath/libswiftDispatch.dylib (compatibility version 1.0.0, current version 4.40.2)
	@rpath/libswiftFoundation.dylib (compatibility version 1.0.0, current version 20.0.0)
	@rpath/libswiftObjectiveC.dylib (compatibility version 1.0.0, current version 1.0.0, weak)
```
I download the xcframework from here
[here](https://github.com/ReactiveX/RxSwift/releases/tag/6.2.0)

I try several versions of the RxSwift (6.1.0,6.0.0rc,...) and try different Polar SDK versions (3.2,3.1,...) but the RxSwift with the  otool return the same version 0.0.0 and the PolarBleSDK wants 1.0.0 or later...
I got the xcframework for the SDK and RxSwift,the build success but the app is crash

thanks in advance

**Comment by JOikarinen** [comment]
@avik-efcom thank you for the report, I am able to reproduce your problem and now there is proposed fix available in release 3.2.1. Could you take PolarBleSdk xcframework from release 3.2.1 and retest you issue?

**Comment by avik-efcom** [comment]
Thanks
